packages/core/types/index.js

```diff
@@ -257,31 +260,31 @@ export type InitialParcelOptions = {|
   +env?: EnvMap,
   +targets?: ?(Array<string> | {+[string]: TargetDescriptor, ...}),
 
-  +disableCache?: boolean,
+  +disableCache?: boolean, // TODO: Change to shouldDisableCache or shouldEnableCache?
   +cacheDir?: FilePath,
-  +killWorkers?: boolean,
   +mode?: BuildMode,
-  +minify?: boolean,
-  +scopeHoist?: boolean,
-  +sourceMaps?: boolean,
-  +publicUrl?: string,
-  +distDir?: FilePath,
-  +hot?: ?HMROptions,
-  +contentHash?: boolean,
-  +serve?: InitialServerOptions | false,
-  +autoinstall?: boolean,
+  +hmrOptions?: ?HMROptions,
+  +shouldContentHash?: boolean,
+  +serveOptions?: InitialServerOptions | false,
+  +shouldAutoInstall?: boolean,
   +logLevel?: LogLevel,
-  +profile?: boolean,
-  +patchConsole?: boolean,
+  +shouldProfile?: boolean,
+  +shouldPatchConsole?: boolean,
 
   +inputFS?: FileSystem,
   +outputFS?: FileSystem,
   +workerFarm?: WorkerFarm,
   +packageManager?: PackageManager,
+  +detailedReport?: ?DetailedReportOptions,
```

Comment:
Should this be `detailedReportOptions`? Or maybe just `reportOptions` or `bundleReportOptions` or `cliReportOptions`?

Ideally this would somehow just be sent directly to the CLI reporter. Maybe once we add support for passing reporters to the programmatic API this could be removed from the core options.

Reply:
Keeping this as a temporary solution sounds good, not sure if we should worry about the name if it's temporary?